Gui und String-Methoden

dgarmann opened the pull request:
# Gui und String-Methoden
- Gui für Ziffern, RGB, Figuren, Kurve
- toString
- parse-Methoden

olheem reviewed (changes requested):
List.java und MNIST-Dateien entfernen!

dgarmann committed:
Gui und String-Methoden

# Gui und String-Methoden
- Gui für Ziffern, RGB, Figuren, Kurve
- toString
- parse-Methoden

olheem reviewed (approved):
Ergänzungen werden hinzugefügt.
